contracts/CrossChain/Strategy2Crv.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+// These are the core Yearn libraries
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+
+import "../interfaces/curve.sol";
+import "../interfaces/yearn.sol";
+import "./StrategyCurveBaseUsingGauge.sol";
+
+contract StrategyCurve2Crv is StrategyCurveBaseUsingGauge {
+    /* ========== STATE VARIABLES ========== */
+    // these will likely change across different wants.
+
+    address public curve; // This is our pool specific to this vault. Use it with zap contract to specify our correct pool.
+
+    IERC20 public constant crv =
+        IERC20(0x1E4F97b9f9F913c46F1632781732927B9019C68b);
+
+    // we use these to deposit to our curve pool
+    IERC20 public constant usdc =
+        IERC20(0x04068DA6C83AFCFA0e13ba15A6696662335D5B75);
+    IERC20 public constant dai =
+        IERC20(0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E);
+    IERC20 public constant wftm =
+        IERC20(0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83);
+
+    address public constant voter = 0x72a34AbafAB09b15E7191822A679f28E067C4a16; // sms
+
+    // rewards token info. we can have more than 1 reward token but this is rare, so we don't include this in the template
+    IERC20 public rewardsToken;
+    bool public hasRewards;
+
+    // Target trades
+    address public rewardsTarget;
```

Comment:
Do these targets need to be separate variables? Imo they would always be the same, because it's the ideal token to deposit to the pool

Reply:
fair! will unify. `tradeTarget`?